src/shared/util/email-domain-validation.ts

```diff
@@ -0,0 +1,12 @@
+import { isEmpty } from 'lodash'
+import validator from 'validator'
+
+export const validateEmailDomains = (emailDomains: string[]): boolean => {
+  return (
+    isEmpty(emailDomains) ||
+    (new Set(emailDomains).size === emailDomains.length &&
+      emailDomains.every((emailDomain) =>
+        validator.isEmail('bob' + emailDomain),
```

Comment:
Can add a comment highlighting that the addition of `bob` is to transform it into an email-like string to test validation against :D

Reply:
Got it!